src/pages/signin/ResendValidationForm.js

```diff
@@ -107,10 +84,25 @@ class ResendValidationForm extends React.Component {
                         {this.props.translate('resendValidationForm.weSentYouMagicSignInLink', {login, loginType})}
                     </Text>
                 </View>
-                {!_.isEmpty(this.state.formSuccess) && (
-                    <Text style={[styles.formSuccess]}>
-                        {this.state.formSuccess}
-                    </Text>
+                {successMessage && (
```

Comment:
I feel like this should be extrapolated out into a reusable component, and should build off of what we already have in place for the red brick road message.

Reply:
For sure, I actually don't know what you're referring to by the red brick road message, do you know where this component is used?